lib/src/git_backend.rs

```diff
@@ -54,7 +54,7 @@ pub struct GitBackend {
 
 impl GitBackend {
     fn new(repo: git2::Repository, extra_metadata_store: TableStore) -> Self {
-        let root_commit_id = CommitId::from_bytes([0; 20]);
```

Comment:
nit: this has to be squashed to the previous commit.

Reply:
Ah, yes. Fixed.